realm/src/main/java/io/realm/internal/Group.java

```diff
@@ -55,18 +51,16 @@ public Group() {
         checkNativePtrNotZero();
     }
 
-    protected native long createNative();
-
     public enum OpenMode {
         // Below values must match the values in realm::group::OpenMode in C++
         READ_ONLY(0),
```

Comment:
`ordinal()` depends on the order, which means that if someone changed the order at some point the code would no longer be valid.